zeros:
  fl_caster_rotation_joint: -9.576824857138178e-05
  fl_caster_r_wheel_joint: 0.10503934348692212
  b_caster_rotation_joint: 3.14159224925643
  b_caster_r_wheel_joint: 0.10300531224194032
  fr_caster_rotation_joint: 9.656550617620496e-05
  fr_caster_r_wheel_joint: 0.10555145456779158
  head_1_joint: 1.0072381497749916e-05
  head_2_joint: 7.030458642187654e-05
  head_3_joint: -1.8492486283605558e-05
  sensorring_joint: 0.0006168416326008952
  arm_right_1_joint: 2.0874473067861103
  arm_right_2_joint: 1.474161799268268
  arm_right_3_joint: -2.9388074593446296
  arm_right_4_joint: -1.4962075806061268
  arm_right_5_joint: 0.5245271967529268
  arm_right_6_joint: -1.617664486532144
  arm_right_7_joint: 1.8312071828042527
  arm_left_1_joint: -0.8123487476643598
  arm_left_2_joint: -1.4750187323940303
  arm_left_3_joint: -0.2095284824664123
  arm_left_4_joint: -1.4965477701005891
  arm_left_5_joint: 0.7510530934233977
  arm_left_6_joint: -1.8370555509091204
  arm_left_7_joint: -1.6082781258138814
  gripper_right_finger_1_joint: 6.825372034757748e-05
  gripper_right_finger_2_joint: -0.0001712272475975496
  gripper_left_finger_1_joint: -6.762316934949553e-05
  gripper_left_finger_2_joint: -1.9918255839357357e-05
